tests/python/gaia-ui-tests/gaiatest/apps/system/regions/cards_view.py

```diff
@@ -110,6 +110,33 @@ class Card(PageRegion):
     _close_button_locator = (By.CLASS_NAME, 'close-button')
     _screenshot_view_locator = (By.CLASS_NAME, 'screenshotView')
     _app_icon_locator = (By.CLASS_NAME, 'appIcon')
+    _title_locator = (By.CLASS_NAME, 'title')
+    _subtitle_locator = (By.CLASS_NAME, 'subtitle')
+
+    @property
+    def title(self):
+        return self.root_element.find_element(*self._title_locator).text
+
+    @property
+    def is_centered(self):
+        screen_width = int(self.marionette.execute_script('return window.innerWidth'))
+        left = self.root_element.rect['x']
+        width = self.root_element.rect['width']
+        # center of card should be within 1px of viewport center
+        return 1 >= abs(screen_width / 2 - (left + width / 2))
```

Comment:
That formula is right, but might be a bit hard to understand at the first sight. How about?
```py
expected_center = screen_width / 2
actual_center = left + width / 2
difference_between_centers_in_px = abs(expected_center - actual_center)
return length_between_centers_in_px < 1
```